athena/utils/calculate.py

```
@@ -0,0 +1,214 @@
+        ", ".join(group_by_cols),
+        ", ".join(map(str, thresholds)),
+    )
+    start = timer()
+
+    if not all(col in coverage_data.columns for col in group_by_cols):
+        raise ValueError(
+            f"Specified group_by columns {group_by_cols} not present in"
+            f" dataframe, available columns: {coverage_data.columns}"
+        )
+
+    if not all(type(x) == int for x in thresholds):
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)